en/components/grids_templates/summaries.md

```diff
@@ -228,7 +228,7 @@ interface IgxSummaryResult {
 }
 ```
 and take optional parameters for calculating the summaries.
-See [Custom summaries, which access all data](#custom-summaries-which-access-all-grid-data) and [Custom summaries with localization](#custom-summaries-with-localization) sections below.
+See [Custom summaries, which access all data](#custom-summaries-which-access-all-grid-data) section below.
```

Comment:
This link doesn't work in Tree Grid and Hierarchical Grid topics (it is specific to the Grid topic).